webclient/src/websocket/commands/SessionCommands.ts

```diff
@@ -326,8 +331,10 @@ export class SessionCommands {
 
     webClient.protobuf.sendSessionCommand(sc, raw => {
       if (raw.responseCode === webClient.protobuf.controller.Response.ResponseCode.RespOk) {
+        SessionCommands.updateStatus(StatusEnum.DISCONNECTED, 'Password successfully updated');
         SessionPersistence.resetPasswordSuccess();
       } else {
+        SessionCommands.updateStatus(StatusEnum.DISCONNECTED, 'Password update failed, please try again');
```

Comment:
If we want to show the error message in the modal, we shouldn't also show it here in the server status. We can talk more about this as a follow up.